app/styles/_variables.scss

```diff
@@ -1,5 +1,5 @@
 //Font Family Variables
-$default-font: "Clear Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
+$default-font:"Fira Sans", Helvetica, Arial, sans-serif;
```

Comment:
Interesting! Does this change anything significant by switching from "Clear Sans" to "Fira Sans"?